### PR TITLE
Added: Home page 2

### PR DESCRIPTION
**Pull request by KaustubhM1997**
1) Create the card component and used the map function
2) Moved route to another folder
3) Made nav and footer components universal
4) Removed commented code
5) Made use of CSS variables where they were missing

Live preview - https://nervous-jepsen-33fba8.netlify.app/

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -1,14 +1,12 @@
 import "./home.css";
 import { Link } from "react-router-dom";
-import {Card} from "../../components/Card/card.jsx";
+import { Card } from "../../components/Card/card.jsx";
 import { categories } from "../../backend/db/categories";
 
 const Home = () => {
-
-  const [categoryMen] = categories; //categories is an array of objects
+  // const [categoryMen] = categories; //categories is an array of objects // we would do this only if there was one category and pass categoryMen into category below.
```
Comment (mimansha-swarup) [inline src/pages/Home/Home.jsx:7]: remove unnecessary comment

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -27,62 +25,19 @@ const Home = () => {
...
-            />
-          </div> */}
+        <div className="card-container">
+          {categories.map(
+            // we take the categories from backend
+            (
+              categor //categor is every element from the categories database
+            ) => (
+              <Card key={categor._id} category={categor} /> //we use key as categor.id as categories (backend has a unique id). We take all categor elements we get from the database and put them in an object which we store in category. Then we pass category to our card component and then destructure it into categoryName and categoryId (these keys are present in categories). Then we just render the destrcutured items as JSX.
+            )
```
Comment (mimansha-swarup) [inline src/pages/Home/Home.jsx:38]: I liked how you have  added meaningful comments
great Work

File: src/main.css [hunk 3]
```diff
@@ -1,143 +1,116 @@
...
-    overflow: hidden;
-    position: fixed;
-    z-index: 1;
-    width: 100%;
-    left: 0;
-    top: 0;
+  display: flex;
+  padding: 1rem;
+  background-color: var(--primary-color);
+  color: white;
```
Comment (mimansha-swarup) [inline src/main.css]: using variable for this will be good practice

File: src/components/Nav/nav.jsx [hunk 4]
```diff
@@ -1,55 +1,45 @@
...
-
-    )
-}
-
-
-
-
+        <div className="badge-element nav-item">
+          <Link to="/cart-management">
+            <i className="fa-solid fa-cart-arrow-down icon"></i>
```
Comment (mimansha-swarup) [inline src/components/Nav/nav.jsx:35]: suggestion: can use react Icon

**Review by mimansha-swarup — APPROVED** [review]
few suggestions rest all looks good 👍

**Review by srishti-maurya — APPROVED** [review]

**Pull request merged**